Sources/QRCodeType.swift

```diff
@@ -68,8 +68,10 @@ struct QRCodeType {
 
 extension QRCodeType {
     /// Get the type by string length
-    static func typeNumber(of text: String, errorCorrectLevel: QRErrorCorrectLevel) throws -> Int {
-        let textLength = text.utf8.count
+    static func typeNumber(of text: String, errorCorrectLevel: QRErrorCorrectLevel, encoding: String.Encoding = .utf8) throws -> Int {
```

Comment:
Maybe move the encoding parameter closer to the text param like this?

```suggestion
    static func typeNumber(of text: String, encoding: String.Encoding = .utf8, errorCorrectLevel: QRErrorCorrectLevel) throws -> Int {
```

Reply:
Good idea, string next to encoding is more concise.